src/BackgroundCells.js

```diff
@@ -18,6 +19,7 @@ import Selection, { getBoundsForNode, isEvent } from './Selection'
 class BackgroundCells extends React.Component {
   static propTypes = {
     date: PropTypes.instanceOf(Date),
+    nowAccessor: PropTypes.func.isRequired,
```

Comment:
i missed a bit of the naming discussion but I think `accessor` is misleading since it's not, its only a function that returns a date. `getNow` sounds a bit more accurate to my ears

Reply:
Especially since we're dropping the `get()` semantics. Will rename to `getNow()`